src/pages/en/guides/upgrade-to/v2.mdx

```diff
@@ -0,0 +1,405 @@
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # Upgrade to Astro v2.x
+  pnpm install astro@latest
+
+  # Example: upgrade React and Tailwind integrations
+  pnpm install @astrojs/react@latest @astrojs/tailwind@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # Upgrade to Astro v2.x
+  yarn add astro@latest
+  
+  # Example: upgrade React and Tailwind integrations
+  yarn add @astrojs/react@latest @astrojs/tailwind@latest
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+## Astro 2.0 Breaking Changes
+
+Astro v2.0 includes some breaking changes, as well as the removal of some previously deprecated features. If your project doesn't work as expected after upgrading to v2.0, check this guide for an overview of all breaking changes and instructions on how to update your codebase.
+
+See [the changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) for full release notes.
+
+### Removed: Support for Node 14
+
+Node 14 is scheduled to reach its End of Life in April 2023. 
+
+Astro 2.0 drops Node 14 support entirely, so that all Astro users can take advantage of Node's more modern features.
+
+#### What should I do?
+
+ Check that both your development environment and your deployment environment are using **Node.js `16.12.0` or later**.
+
+1. Check your local version of node using:
+
+    ```sh
+    node -v
+ 
+    ```
+    If your local development environment needs upgrading, [install Node.js](https://nodejs.org/en/download/).
+
+
+2. Check your [deployment environment's](/en/guides/deploy/) own documentation to verify that they support Node 16. 
+
+    You can specify Node `16.12.0` for your Astro project either in a dashboard configuration setting, or a `.nvmrc` file.
+
+
+### Reserved: `src/content/`
+
+Astro 2.0 now includes the Collections API for organizing your Markdown and MDX files into [content collections](/en/guides/content-collections/). This API reserves `src/content/` as a special folder.
+
+#### What should I do?
+
+Rename an existing `src/content/` folder to avoid conflicts. This folder, if it exists, can now only be used for [content collections](/en/guides/content-collections/).
+
+### Changed: `Astro.site` trailing slash
+
+In 1.x, Astro ensured the URL you set as `site` in `astro.config.mjs` always had a trailing slash when accessed using `Astro.site`.
+
+Astro 2.0 no longer modifies the value of `site`. `Astro.site` will use the exact value defined, and a trailing slash must be specified if desired.
+
+#### What should I do?
+
+ In `astro.config.mjs`, add a trailing slash to the URL set in `site`. 
+
+```js del={5} ins={6}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  site: 'https://example.com',
+  site: 'https://example.com/',
+});
+```
+
+### Changed: `_astro/` folder for build assets
+
+In 1.x, assets were built to various locations, including `assets/`, `chunks/` and the root of the build output.
```

Comment:
The sentence structure felt weird to me in the last bit, I'm wondering if there's a way to make it more seamless.

```suggestion
In 1.x, assets were built to various locations, including `assets/`, `chunks/`, and to the root of the build output.
```

Probably a "just me" thing being non-native, feel free to ignore this one if you feel it's actually ok.